src/devices/scmplus.c

```diff
@@ -0,0 +1,198 @@
+/** @file
+    ERT SCM+ sensors.
+
+    Copyright (C) 2020 Peter Shipley <peter.shipley@gmail.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include <arpa/inet.h>
```

Comment:
Can we get rid of this ?

Reply:
your right I should have used `htobe16` from  `endian.h `?   
although I find `ntohs()` to be more portable.

I would rather not automatically make assumptions on local byte order with  hard a coded
`((t_16 & 0xFF00) >> 8 | (t_16 & 0x00FF) << 8)`

with intermixed with :
`#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__`

Reply:
I'd say we are used to pick bit and bytes from random places in the decoders, for just two instances of short with not real performance requirement just pick t_16 apart and just bitshift-or.

Reply:
ok...
bitshift-or  manually 
Done